docs/SPRINT_PLAN_TEMPLATE.md

```
@@ -0,0 +1,318 @@
+## 🤖 AI Sprint Management Prompt
+
+Use this prompt when closing the previous sprint and kicking off the next one:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)